tut-create-video.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: How do I create a video of my trace?
+sidebar: Create a Video?
+order: 70
+permalink: /tutorials/createvideo
+parent: tutorials
+---
+
+GAPID has the ability to export a captured trace as a video.
+
+This feature is only available via the command line tool `gapit`.
+
+### Prerequisites
+* A previously captured trace file ending in *.gfxtrace
+* Either avconv or ffmpeg encoding libraries on your path (for regular video encoding)
+
+<div class="callouts" markdown="block">
```

Comment:
No need for callouts here - they're for pointing out things in an image.

Reply:
ah, fixed.